web_src/css/helpers.css

```diff
@@ -269,6 +270,17 @@ Gitea's private styles use `g-` prefix.
 .gt-font-17 { font-size: 17px !important }
 .gt-font-18 { font-size: 18px !important }
 
+@media (max-width: 575.98px) {
+  .gt-ml-auto-small { margin-left: auto !important; }
+
+  .gt-o-0-small { order: 0 !important; }
```

Comment:
The code below "@media (max-width: 767.98px) " is already a hacky patch and it already cases some problems.

I do not think it is maintainable to introduce more such patches. And, the comment said `gt` prefixes are doing the best to match tailwind styles, so, it's better to avoid non-tailwind styles using such prefix.

Reply:
Do you have any suggestions on how to do it instead? I don't think it is a good idea to write identical CSS classes for every item to be ordered. And sadly CSS doesn't allow to remove properties depending on size.
I could change the naming to match tailwind thought.....